projects/igniteui-angular/src/lib/services/overlay/position/global-container-position-strategy.ts

```diff
@@ -0,0 +1,28 @@
+import { IPositionStrategy } from './IPositionStrategy';
+import { PositionSettings, HorizontalAlignment, VerticalAlignment, Size, Util } from './../utilities';
+import { fadeIn, fadeOut } from '../../../animations/main';
+import { GlobalPositionStrategy } from './global-position-strategy';
+import { basename } from '@angular-devkit/core';
+
+/**
+ * Positions the element inside outlet based on the directions passed in trough PositionSettings.
+ * These are Top/Middle/Bottom for verticalDirection and Left/Center/Right for horizontalDirection
+ */
+export class GlobalContainerPositionStrategy extends GlobalPositionStrategy {
```

Comment:
I'm thinking just `ContainerPositionStrategy` cuz Global doesn't sound right for what it does

Reply:
And yeah, I'm aware there's one with the same name inside the Grid which is not public. If those two clash, rename the internal one.